bitcoin/src/blockdata/transaction.rs

```diff
@@ -773,7 +773,7 @@ impl Transaction {
     #[inline]
```

Comment:
Why this commit? We have a CI job for this.

Reply:
Running `cargo +nightly fmt` before the CI weekly job runs formats all files yet to be formatted by the CI job.

Reply:
Then just don't do it or only commit relevant changes.